graphistry/feature_utils.py

```diff
@@ -135,29 +173,32 @@ def assert_imported():
 #
 #      _featurize_or_get_edges_dataframe_if_X_is_None
 
-FeatureEngineConcrete = Literal["none", "pandas", "dirty_cat", "torch"]
+FeatureEngineConcrete = Literal["none", "pandas", "dirty_cat", "torch", "cu_cat", "cu_cat|torch"]
```

Comment:
it sounds like we're stuck on a design decision here, or do you have a strong view you can explain already?

I see `"cu_cat|torch"` here, but the rest of the code isn't using